### ISSUE TITLE
Enable downloads

### ISSUE THREAD

**Issue by mlgamble** [satackey/action-google-drive#3]
My workflow is that I want to edit content in Google Drive and that gets downloaded by the action and included in my build.

Specifically, I'm using Gatsby to generate static site content from markdown and I want to store the markdown in Google Drive and have it pulled automatically

**Comment by satackey** [comment]
It looks something like this, doesn't it?

```yaml
- uses: satackey/action-google-drive@v1
  with:
    skicka-tokencache-json: ${{ secrets.SKICKA_TOKENCACHE_JSON }}
    download-from: /foo
    donwload-to: ./bar
    google-client-id: ${{ secrets.GOOGLE_CLIENT_ID }}
    google-client-secret: ${{ secrets.GOOGLE_CLIENT_SECRET }}
```

**Comment by mlgamble** [comment]
Yep! That would work for me

**Comment by satackey** [comment]
I've released [version 1.2.0](https://github.com/satackey/action-google-drive/releases/tag/v1.2.0) with the download enabled.
It is available from the tag `satackey/action-google-drive@v1`.

If this is ok, please close this issue!

**Comment by mlgamble** [comment]
Thanks!
